magprime/models.py

```diff
@@ -2,6 +2,18 @@
 
 
 @Session.model_mixin
+class Group:
+    @presave_adjustment
+    def bucket_pricing_workaround(self):
+        if not self.is_dealer:
+            if self.auto_recalc:
```

Comment:
This is designed to allow us to simply resave groups in order to fix the data. However, it also means that if we check the 'Recalc' checkbox on groups, their cost will always be set to what they owe, so the checkbox is now broken. There wasn't a really a way to make it work, unfortunately, due to how Groups' costs are structured. :(